run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+cd /usr/src/app
+case ${DOCKER_STATE} in
+migrate)
+    echo "running migrate"
+    bundle exec rake db:migrate
+    ;;
+seed)
+    echo "running seed"
+    bundle exec rake db:migrate
+    bundle exec rake db:seed
+    ;;
+vagrant)
+    echo "running seed"
+    bundle exec rake db:create
+    bundle exec rake db:seed
+    bundle exec rake db:migrate
+    ;;
```

Comment:
Is this supposed to be in a different order (`create`, `seed`, `migrate` vs `create`, `migrate`, `seed`) to the `create` command?